src/Header/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './Header.css';
+
+const Header = () => (
```

Comment:
header and footer components could be on App component level (I mean in App folder), but not critical